src/com/magento/idea/magento2plugin/inspections/xml/ModuleDeclarationInModuleXmlInspection.java

```diff
@@ -27,6 +28,8 @@
 
 public class ModuleDeclarationInModuleXmlInspection extends XmlSuppressableInspectionTool {
 
+    private static final String MODULE_NAME = "name";
```

Comment:
add this constant to `com.magento.idea.magento2plugin.magento.files.ModuleXml`
And write a test please. Thank you!